src/cross.js

```diff
@@ -164,7 +164,7 @@ const asResultAccount = (unsafe: mixed): AccountData => {
     index,
     balance,
     // $FlowFixMe No idea how to make flow like this
-    swapHistory: invalidSwapHistory ? [] : swapHistory,
+    swapHistory: swapHistory || invalidSwapHistory ? [] : swapHistory,
```

Comment:
I don't understand the logic. Did you mean `!swapHistory` ?

Reply:
Let's review tomorrow. I would prefer we satisfy flow, maybe by building up an array, and instead of using a flag, I feel we can simply use that same array to set it to [] or something

Reply:
This is indeed meant to be `!swapHistory` and it does not fix flow, because flow is complaining about the array being mixed and not valid 🤷‍♂️

Reply:
ok yeah, that's why we need to build up an array